lib/Cli.php

```diff
@@ -456,7 +456,7 @@ protected function convert($vObj)
      */
     protected function color($vObj)
     {
-        fwrite($this->stdout, $this->serializeComponent($vObj));
+        $this->serializeComponent($vObj);
```

Comment:
I am not familiar with what this `color()` method was supposed to do.
Was it supposed to write output? Or?

Reply:
Well you tell me :-) It appears this function implements the `bin/vobject color` command which can output a color highlighted version of a vobject file such as a vcard. However, the called serializeComponent() function already writes to stdout by itself, and returns nothing, so the write on void that was previously there just makes no sense.

Reply:
Ahhh - the return value of `serializeComponent` is void! That had not got through to my brain. Now I understand why the `fwrite()` wrapping this makes no sense.